VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/models/participant-panel-model.ts

```diff
@@ -23,7 +23,7 @@ export class ParticipantPanelModel extends IndividualPanelModel {
         return this.isCallable && this.isInHearing();
     }
     get isCallable(): boolean {
-        return this.isWitness || this.isQuickLinkUser;
+        return this.isWitness || this.isQuickLinkUser || this.isInterpreter;
```

Comment:
I feel like we shouldn't include interpreter as a callable participant as it is dependent on who they are interpreting for. I think we need to instead look at the logic in the linked participant panel model.

Reply:
To me the issue looks like it's here in LinkedParticipantPanelModel which is why I understand adding interpreter as callable but I worry that that will behave strangely with non witnesses(I think it might show the dismiss option for them)
```
get isCallableAndReadyToJoin(): boolean {
    return this.participants.every(p => p.isCallableAndReadyToJoin);
}
```

Maybe this should check if any are callable and all are available instead of checking each isCallableAndReadyToJoin.